lib/exercism/tooling_job.rb

```diff
@@ -55,7 +55,7 @@ def respond_to_missing?(meth, include_all = true)
       data.key?(meth) || super
     end
 
-    def method_missing(meth)
+    def method_missing(meth, *args)
```

Comment:
Is `*args` used anywhere or did you need to add this?

Reply:
I needed to add it because I discovered if a method is called with args, but the method doesn't exist, rather than a proper exception being raise, a "Incorrect number of args" exception is raised, which is really confusing in consumers of the lib.